Jellyfin.Data/Enums/MediaSegmentAction.cs

```diff
@@ -0,0 +1,33 @@
+namespace Jellyfin.Data.Enums.MediaSegmentAction
+{
+    /// <summary>
+    /// An enum representing the Action of MediaSegment.
+    /// </summary>
+    public enum MediaSegmentAction
+    {
+        /// <summary>
+        /// Auto, use default for type.
+        /// </summary>
+        Auto = 0,
```

Comment:
Isn't the whole point of the actions that the server sends a recommendation for what to do? Why should the server then tell us "figure it out yourself". If there is a default for each segment type the server should just send that default.